lib/membrane_mp3_mad_plugin/decoder.ex

```diff
@@ -45,6 +62,48 @@ defmodule Membrane.MP3.MAD.Decoder do
     end
   end
 
+  defp skip_id3(id3) do
+    import Bitwise
+
+    # taken from https://github.com/thechangelog/id3vx/blob/4e0349ea5bdb7f8bf8430b224151a7293fccb3fd/lib/id3vx.ex#L436
+    result =
+      case id3 do
+        <<"ID3", 2::integer, _minor::integer, _flags::size(8), tag_size::binary-size(4),
+          content::binary>> ->
+          {:tag, tag_size, content}
+
+        <<"ID3", 3::integer, _minor::integer, _flag_bytes::size(8), tag_size::binary-size(4),
+          content::binary>> ->
+          {:tag, tag_size, content}
+
+        <<"ID3", 4::integer, _minor::integer, _unsynchronisation::size(1),
+          _extended_header::size(1), _experimental::size(1), _footer::size(1), _unused::size(4),
+          tag_size::binary-size(4), content::binary>> ->
+          {:tag, tag_size, content}
+
+        payload when byte_size(payload) < 10 ->
+          :skipping
+
+        # no id3
+        payload ->
+          {:skipped, payload}
+      end
+
+    with {:tag, tag_size, content} <- result do
+      tag_size =
+        tag_size
+        |> :binary.bin_to_list()
+        |> Enum.reverse()
+        |> Enum.with_index()
+        |> Enum.reduce(0, fn {el, index}, acc -> acc ||| el <<< (index * 7) end)
```

Comment:
We can move it to some external function that does that, as it is slightly enigmatic - we can call that function `skip_most_significant_bits` or something like that (as I believe that is what it does :D )